addons/packages/pinniped/0.12.1/bundle/config/values.star

```diff
@@ -10,16 +10,22 @@ def validate_pinniped():
   data.values.tkg_cluster_role in ("management", "workload") or assert.fail("tkg_cluster_role must be provided to be either 'management' or 'workload'")
   data.values.infrastructure_provider in ("vsphere", "azure", "aws") or assert.fail("infrastructure_provider must be provided to be either 'vsphere', 'azure' or 'aws'")
   if data.values.identity_management_type:
-    data.values.identity_management_type in ("oidc", "ldap") or assert.fail("identity_management_type must be provided to be either 'oidc' or 'ldap'")
+
+    if is_mgmt_cluster():
+      data.values.identity_management_type in ("oidc", "ldap") or assert.fail("identity_management_type for management clusters must be either 'oidc' or 'ldap'")
+    end
+    if is_workload_cluster():
+      data.values.identity_management_type in ("none", "oidc", "ldap") or assert.fail("identity_management_type for workload clusters be either 'none', 'oidc' or 'ldap'")
+    end
   end
-  if data.values.tkg_cluster_role == "workload":
+  if is_workload_cluster():
```

Comment:
Should this be `render_on_workload_cluster()` rather than `is_workload_cluster()`? If `identity_management_type` is none, then there's no need to validate the issuer or the ca_bundle, right?

Reply:
That seems reasonable, we can make the change quick!